drivers/serial/uart_npcx.c

```diff
@@ -1066,6 +1066,9 @@ static int uart_npcx_init(const struct device *dev)
 	/* Disable all UART tx FIFO interrupts */
 	uart_npcx_dis_all_tx_interrupts(dev);
 
+	/* Disable rx FIFO not empty interrupt */
+	inst->UFRCTL &= ~(BIT(NPCX_UFRCTL_RNEMPTY_EN));
```

Comment:
Equivalent to `uart_npcx_irq_rx_disable(dev)`

Reply:
I agree - re-use the existing helper routine.

Reply:
Done.